src/app/theme/themes/_index.scss

```diff
@@ -0,0 +1,6 @@
+@import 'light.scss';
+@import 'dark.scss';
+@import 'icydark.scss';
+@import 'legends.scss';
+@import 'cake.scss';
+@import 'greenish.scss';
```

Comment:
We need to copy over the cake and greenish themes into their own scss files.

Reply:
that was a big miss, just forgot to commit them.